AdyenComponents/BLIK/BLIKComponentConfiguration.swift

```diff
@@ -20,7 +20,7 @@ public struct BLIKComponentConfiguration: CheckoutComponentConfiguration {
 
     package var style: FormComponentStyle
 
-    package var theme: AdyenTheme = .init()
+    package var theme: CheckoutTheme?
```

Comment:
Why we switch to optional theme?

Reply:
Cause this is using the theme from the `CheckoutComponentConfiguration` which is in `Adyen` and I could not init this with `AdyenTheme` as we not have access of `AdyenUI` in `Adyen`.